docs/DeveloperGuide.adoc

```diff
@@ -341,6 +366,27 @@ Use case ends.
 +
 Use case resumes at step 2.
 
+[discrete]
+=== Use case: Delete module
+
+*MSS*
+
+1. User requests to show module plan
+2. AddressBook shows module plan
+3. User requests to delete a module taken
+4. AddressBook deletes module
```

Comment:
This use case looks nice.
How about if we add the response of the server as: 
5. the AddressBook displays the updated module list after successful deletion.